### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jellySWATy**
Added new popUps,
Added source files,
Fixed functions cards,
Added subscribe handler,
Added videos

### CODE REVIEW

File: build/index.html [hunk 1]
```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <title>Главная страница</title><!--[if IE]>
```
Comment (sefimov) [inline build/index.html]: Use English please

File: build/index.html [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+          <div class="container">
+            <div class="product-info__content">
+              <div class="product-info__left fadeInLeft wow" wow-delay="2s">
+                <div class="product-info__header">
+                  <div class="product-condition">soon</div>
+                  <div class="product-status">Under development</div>
+                </div>
+                <h1 class="main-title">Back to the Office<span class="product-status">Under development</span></h1>
+                <p class="description-text">The application for remote work.</p>
+                <p class="description-text">Back To The Office will help remote employees to become a part of the team.</p>
```
Comment (sefimov) [inline build/index.html]: please change  "Back To **T**he Office" to "Back To **t**he Office"

File: build/index.html [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+                    <label class="form__label" for="email">email</label>
+                    <input class="form__input email" type="email" name="email" id="email" required>
+                  </div>
+                  <label class="contacts-form__textarea-label" for="message">message</label>
+                  <textarea class="contacts-form__textarea" name="message" id="message"></textarea><input type="hidden" name="bottom-form" value="bottom-form">
+                  <input class="g-btn contacts-btn" type="submit" value="send us ⟶">
+                </form>
+              </div>
+              <div class="contacts__content-right fadeInRight wow" wow-delay="2s"></div>
+            </div><a class="write-to-us fadeIn wow" href="mailto:backtotheoffice@gmail.com" wow-delay="2s">
```
Comment (sefimov) [inline build/index.html]: Please change mail to `btto.contact@gmail.com`

File: build/index.html [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+              <svg class="icon icon-mail ">
+                <use href="static/img/svg/symbol/sprite.svg#mail"></use>
+              </svg>
+              <p class="contacts-text">email: backtotheoffice@gmail.com</p></a>
```
Comment (sefimov) [inline build/index.html]: Same here

File: data/navigation.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "navPages": {
+    "Главная страница" : "main.html",
```
Comment (sefimov) [inline data/navigation.json]: Use English please

File: dev/static/js/main.js [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+    };
+    
+    const showMessage = async message => {
+        message.classList.add('active');
+        await delay(4000);
+        message.classList.remove('active');
+    };
+    
+    const ajaxSend = formData => {
+        fetch("http://localhost:8081/api/v1/landing/send", {
```
Comment (sefimov) [inline dev/static/js/main.js]: use relative links instead of absolute

File: dev/static/stylus/modules/modules.styl [hunk 7]
```diff
@@ -0,0 +1,21 @@
+// Шапка
```
Comment (sefimov) [inline dev/static/stylus/modules/modules.styl:1]: Use english please

File: dev/static/stylus/utils/vars.styl [hunk 8]
```diff
@@ -0,0 +1,30 @@
+//ЦВЕТА
```
Comment (sefimov) [inline dev/static/stylus/utils/vars.styl]: Use english please

**Review by sefimov — CHANGES_REQUESTED** [review]

File: build/index.html (hunk 1 above)
Comment (jellySWATy) [reply build/index.html]: fixed

File: build/index.html (hunk 2 above)
Comment (jellySWATy) [reply build/index.html]: fixed

File: build/index.html (hunk 3 above)
Comment (jellySWATy) [reply build/index.html]: fixed

File: build/index.html (hunk 4 above)
Comment (jellySWATy) [reply build/index.html]: fixed

File: data/navigation.json (hunk 5 above)
Comment (jellySWATy) [reply data/navigation.json]: fixed

File: dev/static/js/main.js (hunk 6 above)
Comment (jellySWATy) [reply dev/static/js/main.js]: fixed

File: dev/static/stylus/modules/modules.styl (hunk 7 above)
Comment (jellySWATy) [reply dev/static/stylus/modules/modules.styl:1]: fixed

File: dev/static/stylus/utils/vars.styl (hunk 8 above)
Comment (jellySWATy) [reply dev/static/stylus/utils/vars.styl]: fixed

**Pull request merged**